x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/attachment.tsx

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiCode, EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
+
+import {
+  ComboBoxField,
+  FIELD_TYPES,
+  UseField,
+  ToggleField,
+  Field,
+} from '../../../../../../shared_imports';
+
+import { FieldNameField } from './common_fields/field_name_field';
+import { TargetField } from './common_fields/target_field';
+import { IgnoreMissingField } from './common_fields/ignore_missing_field';
+import { FieldsConfig, to, from } from './shared';
+
+const propertyValues: string[] = [
+  'content',
+  'title',
+  'author',
+  'keywords',
+  'date',
+  'content_type',
+  'content_length',
+  'language',
+];
+
+const fieldsConfig: FieldsConfig = {
+  /* Optional field configs */
+  indexed_chars: {
+    type: FIELD_TYPES.TEXT,
```

Comment:
Shouldn't this be `FIELD_TYPES.NUMBER` to prevent the user from typing in anything different from a number?